src/api/routes/summaries.ts

```diff
@@ -77,6 +79,14 @@ const list = procedure
 
   checkPermissionForGroup(ctx.user, t.group);
 
+  const handlebars = await extractHandlebars(transcriptId);
+  const nameMap = await createNameMap(handlebars);
+  
+  // using Handlebars.js to compile and return summaries with handlebar replaced
```

Comment:
you don't need to use comments to describe WHAT the code does unless the code isn't too self explanatory.